src/components/sbb-selection-panel/sbb-selection-panel.scss

```diff
@@ -0,0 +1,121 @@
+  --sbb-selection-panel-border-color: var(--sbb-color-cloud-default);
+}
+
+:host([data-disabled]:not([data-disabled='false'])) {
+  --sbb-selection-panel-cursor: default;
+  --sbb-selection-panel-badge-background: var(--sbb-color-smoke-default);
+}
+
+:host([disable-animation]:not([disable-animation='false'])) {
+  --sbb-selection-panel-animation-duration: 0s;
+}
+
+:host([data-has-content][force-open]:not([force-open='false'])),
+:host([data-has-content][data-checked]:not([data-checked='false'])) {
+  --sbb-selection-panel-input-padding: var(--sbb-spacing-responsive-xs)
+    var(--sbb-spacing-responsive-xxs) var(--sbb-spacing-responsive-xxs)
+    var(--sbb-spacing-responsive-xxs);
+  --sbb-selection-panel-content-visibility: visible;
+  --sbb-selection-panel-content-max-height: calc(
+    var(--sbb-selection-panel-content-height) + var(--sbb-spacing-responsive-xxs) +
+      var(--sbb-spacing-responsive-xs)
+  );
+  --sbb-selection-panel-content-opacity: 1;
+  --sbb-selection-panel-content-padding-block-end: var(--sbb-spacing-responsive-xs);
+  --sbb-selection-panel-content-transition: max-height var(--sbb-selection-panel-animation-duration)
+      var(--sbb-animation-easing),
+    padding var(--sbb-selection-panel-animation-duration) var(--sbb-animation-easing),
+    opacity var(--sbb-selection-panel-animation-duration)
+      var(--sbb-selection-panel-animation-duration) var(--sbb-animation-easing);
+}
+
+.sbb-selection-panel {
+  flex: auto;
+  position: relative;
+  width: 100%;
+  background-color: var(--sbb-selection-panel-background);
+  border: var(--sbb-border-width-1x) solid var(--sbb-selection-panel-border-color);
+  border-radius: var(--sbb-border-radius-4x);
+  transition: {
+    duration: var(--sbb-selection-panel-animation-duration);
+    timing-function: var(--sbb-animation-easing);
+    property: border-color;
+  }
+}
+
+.sbb-selection-panel__badge {
+  user-select: none;
+  pointer-events: none;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  border-radius: calc(var(--sbb-border-radius-4x) - var(--sbb-border-width-1x));
+  overflow: hidden;
+}
+
+.sbb-selection-panel__content {
+  visibility: var(--sbb-selection-panel-content-visibility);
+  max-height: var(--sbb-selection-panel-content-max-height);
+  opacity: var(--sbb-selection-panel-content-opacity);
+  padding-inline: var(--sbb-selection-panel-content-padding-inline);
+  padding-block-end: var(--sbb-selection-panel-content-padding-block-end);
+  transition: var(--sbb-selection-panel-content-transition);
+}
+
+sbb-divider {
+  margin-block-end: var(--sbb-spacing-responsive-xxs);
+}
+
+::slotted(sbb-card-badge) {
+  --sbb-card-badge-background-color-primary: var(--sbb-selection-panel-badge-background);
+}
+
+::slotted(sbb-radio-button),
+::slotted(sbb-checkbox) {
+  cursor: var(--sbb-selection-panel-cursor);
+  pointer-events: var(--sbb-selection-panel-input-pointer-events);
+  display: block;
+  padding: var(--sbb-selection-panel-input-padding);
```

Comment:
could it be done with margin?

Reply:
Ok I see. just want to minimize impact / amount of styles on a host. other option would be with data attributes. But maybe we can live with the current solution? Don't know :D